src/ice/agent.rs

```diff
@@ -678,9 +679,9 @@ impl IceAgent {
     /// This is done for host candidates disappearing due to changes in the network
     /// interfaces like a WiFi disconnecting or changing IPs.
     ///
-    /// Returns `true` if the candidate was found and invalidated.
+    /// Returns `Ok()` if we invalidated a candidate or `Err` if the candidate was unknown to us.
```

Comment:
There are several ways of modelling this API. I am not particularly set on this one. It felt idiomatic and I wanted to get some feedback on the overall idea.